src/azure-cli/azure/cli/command_modules/acr/_format.py

```diff
@@ -93,6 +93,55 @@ def agentpool_output_format(result):
     return _output_format(result, _agentpool_format_group)
 
 
+def connected_registry_output_format(result):
+    return _output_format(result, _connected_registry_format_group)
+
+
+def connected_registry_list_output_format(result):
+    family_tree = {}
+    for reg in result:
+        parent_id = _get_value(reg, 'parent', 'id')
+        parent_name = '' if parent_id.isspace() else parent_id.split('/connectedRegistries/')[1]
+        name = ' ' + _get_value(reg, 'name')
+        family_tree[_get_value(reg, 'id')] = {
+            "name": name,
+            "id": _get_value(reg, 'id'),
+            "connectionState": _get_value(reg, 'connectionState'),
```

Comment:
is `conectionState` value used? if used, is the name supposed to be connection_state?

Reply:
that's just the key name used. I prefer not to change so the code is easier to follow. 
As for the the title name printed, that is in line 315 and it was the last name suggested in the meeting since it was mentioned to use status as misleading. 
If you have any other suggestion, let me know. I'll change the tittle name.